### ISSUE TITLE
Search function broken

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/panel#7574]
When trying to use search function on panel web site its starts some infinite loop "Service Worker"?

https://github.com/user-attachments/assets/3e51994e-b82a-47f3-bafb-df0af9181426

I've tried a hard refresh of the web site, but it does not help me.

**Comment by ahuang11** [comment]
Does it work if you press enter?

I can't reproduce the infinite workers issue, and I can still search.

<img width="1486" alt="Image" src="https://github.com/user-attachments/assets/b16f837c-b8bc-4e35-98ea-f2ba971d1146" />

**Comment by MarcSkovMadsen** [comment]
No.

The problem is my browser is in a state that often renders the Panel web site worthless. And I cannot figure out how to reset it. Other users might be in the same situation.

**Comment by philippjfr** [comment]
I don't remember hearing about this before, I also can't reproduce.

**Comment by philippjfr** [comment]
Would be helpful if you could go into your browser console, go to Application and then show the output you see under "Service Worker"

**Comment by MarcSkovMadsen** [comment]
Here is a non-search example. Just opening the page

https://github.com/user-attachments/assets/a319d859-125d-4b66-8e7c-ec559e289cac

I have many serviceworkers installed @philippjfr. There are more if I scroll down. Is that a problem?

![Image](https://github.com/user-attachments/assets/a39a99f2-7709-4124-8734-a3337b61b89e)

**Comment by philippjfr** [comment]
Yes they should always replace each other, so there should only be one for panel.holoviz.org
